plugins/steve/resource-instance.js

```diff
@@ -437,6 +437,18 @@ export default {
     };
   },
 
+  getAnnotationValue() {
+    return (label) => {
+      return get(this, ['metadata', 'annotations', label]);
+    };
+  },
+
+  getLabelValue() {
+    return (label) => {
+      return get(this, ['metadata', 'labels', label]);
+    };
+  },
```

Comment:
I think if these methods were used in a computed property it wouldn't correctly recompute when the label/annotation changes.  You don't actually use them that way, but having it around is still unsafe if it doesn't work.  So the calls to this should probably just be directly `this.metadata?.annotations?.[label]` instead.